src/theme.tsx

```diff
@@ -167,4 +168,5 @@ export type Theme = {
     isIos: boolean;
     useHrefDecorator: () => (href: string) => string;
     t: (token: TextToken) => string;
+    preventCopyInFormFields: boolean;
```

Comment:
this allows configuring the blocking of copy in form fields for the whole app (required in O2DE/Blau Novum apps)

Reply:
remember to update the related doc (theme-config.md)

btw, I see that I forgot to update it with the translations changes :/

Reply:
Added. Also edited the doc related with texts